aea/runtime.py

```diff
@@ -346,7 +346,7 @@ async def run_runtime(self) -> None:
         """Run runtime which means start agent loop, multiplexer and storage."""
         self._state.set(RuntimeStates.starting)
         await asyncio.gather(
-            self._start_multiplexer(), self._start_agent_loop(), self._start_storage()
+            self._start_agent_loop(), self._start_multiplexer(), self._start_storage()
```

Comment:
why the change of order?

Reply:
actually was done during some exploration, nothing changed actually.

was looking for unreachable code and wondered is order important. can be reverted, or not.

Reply:
okay, it's fine then

Reply:
@solarw since we don't 100% know that this won't affect some edge case in prod I'd say we retain the original order

Reply:
@DavidMinarsch restored